utils/zerovec/src/ule/unvalidated.rs

```diff
@@ -412,6 +419,28 @@ impl<'de> serde::Deserialize<'de> for UnvalidatedChar {
     }
 }
 
+#[cfg(feature = "databake")]
+impl databake::Bake for UnvalidatedChar {
+    fn bake(&self, env: &databake::CrateEnv) -> databake::TokenStream {
+        match self.try_to_char() {
+            Ok(ch) => {
+                env.insert("zerovec");
+                let ch = ch.bake(env);
+                databake::quote! {
+                    zerovec::ule::UnvalidatedChar::from_char(#ch)
+                }
+            }
+            Err(_) => {
+                env.insert("zerovec");
+                let u24 = u32::from_le_bytes([self.0[0], self.0[1], self.0[2], 0]);
```

Comment:
Is there no `bake` here intentionally?

Reply:
Somewhat, it's more code for the same result.